python/ray/data/tests/test_operators.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+from typing import List, Iterable, Any
+import time
+
+import ray
+from ray.data.block import Block
+from ray.data._internal.execution.interfaces import RefBundle, PhysicalOperator
+from ray.data._internal.execution.operators.all_to_all_operator import AllToAllOperator
+from ray.data._internal.execution.operators.map_operator import MapOperator
+from ray.data._internal.execution.operators.input_data_buffer import InputDataBuffer
+from ray.data._internal.execution.util import make_ref_bundles
+from ray.tests.conftest import *  # noqa
+from ray._private.test_utils import wait_for_condition
+
+
+def _get_blocks(bundle: RefBundle, output_list: List[Block]):
+    for (block, _) in bundle.blocks:
```

Comment:
Nit: Parentheses are redundant.
```suggestion
    for block, _ in bundle.blocks:
```